tools/src/process-allure-reports.py

```diff
@@ -17,17 +17,25 @@ def parse_args():
     parser.add_argument('--cid', required=True, type=str, help='Container ID')
     parser.add_argument('--run_id', required=True, type=str, help='GitHub run ID')
     parser.add_argument('--allure_report', type=str, help='Path to generated allure report directory',
-                        default='allure_report')
+                        default='allure_report'),
+    parser.add_argument('--expire', type=int,
```

Comment:
so basically that is `--expire_after`? also, you are able to make it in hours, not epochs: get epoch duration and block longevity with `neofs-cli`. not sure if it is required

Reply:
> make it in hours, not epochs

Can you please give me an example?

Reply:
epoch equals an hour cause it is `Time per block` * `Epoch duration` seconds:

```
▶ neofs-cli netmap netinfo -r grpcs://st1.storage.fs.neo.org:8082                                        
...
Time per block: 15s
NeoFS network configuration (system)
...
  Epoch duration: 240
...
```

that info is always true and available. it can prevent from `(hours for current testnet and mainnet)` comments to script args (if we need it of course)

Reply:
> epoch equals an hour cause it is `Time per block` * `Epoch duration` seconds:
> 
> ```
> ▶ neofs-cli netmap netinfo -r grpcs://st1.storage.fs.neo.org:8082                                        
> ...
> Time per block: 15s
> NeoFS network configuration (system)
> ...
>   Epoch duration: 240
> ...
> ```
> 
> that info is always true and available. it can prevent `(hours for current testnet and mainnet)` comments to script args (if we need it of course)

Ok, I'll fix the description of the argument.
But anyway, I think it's better to send the epoch to the script.
I calculate the expiration time in the workflow file:
https://github.com/nspcc-dev/neofs-node/pull/2511

Reply:
Renaming to `--expires-at` or something (see nspcc-dev/neofs-node#1574) can help. But epochs are OK here, hour->epoch mapping likely can more easily be done by a layer above.

Reply:
> Renaming to `--expires-at` or something (see [nspcc-dev/neofs-node#1574](https://github.com/nspcc-dev/neofs-node/issues/1574)) can help. But epochs are OK here, hour->epoch mapping likely can more easily be done by a layer above.

Done

Reply:
> @carpawell, are we OK with expires-at here?

totally